examples/jsm/objects/LensflareGPU.js

```diff
@@ -0,0 +1,321 @@
+import {
+	AdditiveBlending,
+	Box2,
+	BufferGeometry,
+	Color,
+	FramebufferTexture,
+	InterleavedBuffer,
+	InterleavedBufferAttribute,
+	Mesh,
+	MeshBasicNodeMaterial,
+	NodeMaterial,
+	UnsignedByteType,
+	Vector2,
+	Vector3,
+	Vector4 } from 'three';
+
+import { texture, textureLoad, uv, ivec2, vec2, vec4, positionGeometry, reference, varyingProperty, materialReference, Fn, Node } from 'three/tsl';
+
+class Lensflare extends Mesh {
```

Comment:
According to https://github.com/mrdoob/three.js/pull/29019#issuecomment-2259786334, we might want to rename the module to `LensflareMesh`.

Reply:
sounds reasonable.